tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs

```diff
@@ -86,6 +87,33 @@ public async Task GetAccessTokenForApp_WithUserScope_MsalServiceExceptionThrownA
             Assert.Equal(0, _msalTestTokenCacheProvider.Count);
         }
 
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        [InlineData("some_secret")]
+        public void ApplicationOptionsIncludeClientSecret(string clientSecret)
+        {
+            // Arrange
+            InitializeTokenAcquisitionObjects();
```

Comment:
I don't think we need this line. 
Plus seems like this whole test method belongs to unit tests and not integration.